__tests__/helpers/openingHoursHelper.test.js

```diff
@@ -0,0 +1,215 @@
+import { getReadableDay, isOpen } from '../../src/helpers';
+
+const todayString = getReadableDay(new Date());
+
+const FAKE_NOW_DATE = new Date();
```

Comment:
i think it would be more clear to get the weekday from the time that is checked in the methods, in this case `const weekday = getReadableDay(FAKE_NOW_DATE)`. i know at the end it will result in the same, but for readability and context, you know? and with using `weekday` variable we could get rid of every `todayString` in jsons and just use `weekday`, right?!